code/game/objects/items/stacks/tiles/tile_types.dm

```diff
@@ -97,6 +97,19 @@
 				if(T.canBuildPlating(S) == BUILD_SUCCESS)
 					qdel(L)
 
+/obj/item/stack/tile/wood/attackby(obj/item/weapon/W as obj, mob/user as mob)
+	if(iswrench(W))
+		if(amount >= 4)
```

Comment:
change this to if(use(4)) and remove the use(4) later down - the sanity is handled in there.